pldp/agency/models.py

```diff
@@ -0,0 +1,46 @@
+import uuid
+
+from django.db import models
+from django.utils.translation import ugettext as _
+
+from languages_plus.models import Language
+
+
+class Agency(models.Model):
+    id = models.UUIDField(primary_key=True,
+                          default=uuid.uuid4,
+                          editable=False)
+    name = models.CharField(max_length=1000,
+                            help_text=_("Full name of the agency that is "
+                                        "conducting/posting the study."))
+    department = models.CharField(max_length=1000,
+                                  null=True,
```

Comment:
First comment, as @beamalsky and I work to complete this! And it's a picky technical one. [Let's follow Django convention and remove `null=True` for CharFields.](https://docs.djangoproject.com/en/2.1/ref/models/fields/#null)